THANKS.txt

```diff
@@ -242,6 +242,7 @@ Mark Borgerding for contributing linalg.convolution_matrix.
 Shashaank N for contributions to scipy.signal.
 Frank Torres for fixing a bug with solve_bvp for large problems.
 Ben West for updating the Gamma distribution documentation.
+Will Tirone for contributing to scipy.stats
```

Comment:
Can you remove this line?

Reason being that adding to the thanks.txt file is a common cause for git rebase conflicts. 
We've been experimenting with other approaches, for 1.5.0 release we had a wiki page that we asked people to add themselves to you can see it here: 
https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.5.0*

I've initiated a discussion on the scipy-dev list and seems there is support for changing the acknowledgments mechanism. We may move the thanks on this file to a separate wiki page (that we add to w/each release) or we may migrate things to this page: https://scipy.org/scipylib/donations.html (still adding on each release)
decision on which is still TBD. 

We'll make sure you're given proper credit though so don't worry about that here.